games/b.yaml

```diff
@@ -309,7 +309,25 @@
   updated: 2018-12-27
   video:
     youtube: N1pGgX_rFOQ
-
+    
+- name: 'BOOM: Remake'
+  images:
+  - https://silverweed.github.io/assets/img/boom/boom_screen1.png
+  - https://silverweed.github.io/assets/img/boom/boom_screen2.png
+  lang: C++
+  license:
+  - custom
+  framework:
+  - SFML
+  content: open
+  development: sporadic
```

Comment:
```suggestion
  development: active
```

Reply:
From https://github.com/silverweed/lifish/commits/boom, there have been many commits this month